package/base-files/files/etc/init.d/led

```diff
@@ -49,6 +51,11 @@ load_led() {
 	[ -e /sys/class/leds/${sysfs}/brightness ] && {
 		echo "setting up led ${name}"
 
+
+		read max_brightness </sys/class/leds/${sysfs}/max_brightness
+		[ "$brightness" -lt "0" ] || [ "$brightness" -gt "$max_brightness" ] &&
```

Comment:
is '0' a string or an integer? quoted numbers are treated as strings :)

```suggestion
		[ "${brightness}" -lt 0 ] || [ "${brightness}" -gt "${max_brightness}" ] &&
```

Reply:
fixed